internal/common/retry.go

```diff
@@ -8,12 +8,17 @@ import (
 	"time"
 
 	backoff "github.com/cenkalti/backoff/v4"
+	"github.com/elastic/e2e-testing/internal/shell"
 )
 
 // TimeoutFactor a multiplier for the max timeout when doing backoff retries.
 // It can be overriden by TIMEOUT_FACTOR env var
 var TimeoutFactor = 3
 
+func init() {
+	TimeoutFactor = shell.GetEnvInteger("TIMEOUT_FACTOR", TimeoutFactor)
+}
```

Comment:
Now that I see this together. Even better, this should work too, so `init()` is not needed.
```
const defaultTimeoutFactor = 3
var TimeoutFactor = shell.GetEnvInteger("TIMEOUT_FACTOR", defaultTimeoutFactor)
```